server/Server.cpp

```diff
@@ -159,5 +165,6 @@ void Server::checkConnectionsForTimeout() {
 }
 
 Server::~Server() {
+	std::cout << "IM DEAD" << std::endl;
```

Comment:
Not sure if this is a remainer from debugging, but when we print a message in the deconstructor, would be nice to get information which server is deconstructed, for example by printing the ip adress

Reply:
was for debugging, i removed it for now.